app/src/main/java/com/simplecity/amp_library/playback/MusicService.java

```diff
@@ -371,7 +371,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
 
                         // Possible solution: (A) Show the Shuttle notification, despite the fact that music isn't playing. Need to customise notification to allow for an empty queue (no current song)
                         // We could try to generate a queue of random songs as well, but there's no guarantee the user has music on their device.
-
+                        newUiEvent(queueManager.getCurrentSong(), UiEventType.SKIP);
```

Comment:
Should we rename `UiEventType.SKIP` to `NEXT` to match the Shuttle implementation? `NEXT` may be easier to understand, and is a better "opposite" to `PREV`.

Also, we're logging the song and seek position that is playing when the user tapped "NEXT" button, right? Not the song that we're skipping to.

Reply:
Yes - You're right. We log the information of the song that is being skipped and the seek position at which the Song was skipped both on the UI and Player event. Also, we're logging the player event `PLAY` of the next song on the queue with seek position 0. 

`UiEventType.NEXT` sounds good to maintain the consistency of variables considering the implementation in Shuttle. Although, we may have to consider any `SKIP` event in the Firestore which was logged while testing.